### PR TITLE
Add support for Razer Naga Trinity

### PR DESCRIPTION
**Pull request by vincenttaglia**
The static color changing is pretty bloated right now because one of the many packets sent before the static color changing command is some sort of mode switcher that needs to be sent after initialization before color changing command payloads are sent or they won't do anything. I'm planning on narrowing it down to the one required command and executing that only on mouse initialization.

### CODE REVIEW

**Comment by vincenttaglia** [comment]
Every feature available with the Naga and programmed here is working as intended. I think there is enough functionality and polish to merge with the main at this point.

**Comment by vincenttaglia** [comment]
I trimmed the fat in the set static effect method, and refactored the code to look more in line with the rest of the program. I think this branch is at a good place to merge with master.

**Comment by RobertDziarnowski** [comment]
What is required to get it merged?

**Comment by vincenttaglia** [comment]
@robdz not sure about what it takes to get merged, but I have instructions on how to compile the working branch yourself [here](https://github.com/openrazer/openrazer/issues/541#issuecomment-531525329).

File: README.md
Comment (z3ntu) [inline README.md:84]: Please move this line one up, to have them sorted

File: driver/razerchromacommon.c
Comment (z3ntu) [inline driver/razerchromacommon.c]: I'm assuming this comment isn't correct so remove it please

File: driver/razermouse_driver.c
Comment (z3ntu) [inline driver/razermouse_driver.c]: Stray semicolon

File: driver/razermouse_driver.c
Comment (z3ntu) [inline driver/razermouse_driver.c:457]: I'm not happy about this section, but I'm not sure where to move it if it's really required :(

**Review by z3ntu — CHANGES_REQUESTED** [review]
See comments

File: driver/razermouse_driver.c
Comment (vincenttaglia) [reply driver/razermouse_driver.c]: At some point the code wouldn't compile without that semicolon but it doesn't seem to be an issue now.

File: driver/razermouse_driver.c
Comment (vincenttaglia) [reply driver/razermouse_driver.c:457]: Yea, it's completely different than other payloads that have established methods. I wasn't sure where to put it either.

File: driver/razerchromacommon.c
Comment (vincenttaglia) [reply driver/razerchromacommon.c]: Done

File: README.md
Comment (vincenttaglia) [reply README.md:84]: Done

**Comment by mateuszkoros** [comment]
Hey, what is the status? I see this PR has been stuck for some time.

**Comment by poirierdag** [comment]
> Hey, what is the status? I see this PR has been stuck for some time.

I 2nd that question, can't wait to be able to use my mouse.

**Comment by vincenttaglia** [comment]
Waiting on a re-review from z3ntu

**Review by z3ntu — APPROVED** [review]
LGTM

I've squashed the commits and added a Co-authored-by tag for you, thank you very much for pushing this forward!

**Pull request merged**
